ctx.go

```diff
@@ -782,6 +782,14 @@ func (c *Ctx) Next() (err error) {
 	return err
 }
 
+// Restart request handling instead of going to next handler. This may be usefull
+// after changing the request path. Note that handlers might be executed again.
+func (c *Ctx) Restart() error {
```

Comment:
can we use a more descriptive name

at first glance you can not see what should be restarted

maybe RestartRouting

Reply:
Sure. I thought of this as an alternative to `return c.Next()`; therefore the shorter name.